### PR TITLE
feat: added the list component.

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
- added two types of lists components

1. normal lists 
2. checked list
lists-preview
s

### CODE REVIEW

File: components/lists.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+.list-page-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.list-container{
+    width: 20rem;
+    height: 15rem;
+    background-color: hsl(var(--primary-lighter));
```
Comment (nishasen) [inline components/lists.css:12]: you can add hsl color code directly in your variable declaration page, just a suggestion.

File: components/lists.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.list{
+    list-style: none;
+    width: 12rem;
+    background-color: white;
```
Comment (nishasen) [inline components/lists.css:22]: Use variable here

File: components/lists.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.list-title{
+    padding-bottom: 5px;
+    border-bottom: solid 1px hsl(var(--gray-color));
+    width: 10rem;
```
Comment (ttahm3d) [inline components/lists.css]: Dont limit the width.

**Review by ttahm3d — APPROVED** [review]
Looks good. have added a suggestion which can be worked later on.

**Review by nishasen — APPROVED** [review]

File: components/lists.css (hunk 2 above)
Comment (vaibhav-mahalle) [reply components/lists.css:22]: made the required changes.

File: components/lists.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply components/lists.css:12]: will keep that in mind

**Pull request merged**